src/appleseed.shaders/src/maya/as_maya_psdFileTex.osl

```diff
@@ -0,0 +1,326 @@
+        string widget = "null"
+    ]]
+)
+{
+    if (in_disableFileLoad || in_fileTextureName == "")
+    {
+        out_outColor = in_defaultColor;
+
+        maya_colorBalance(
+            in_colorGain,
+            in_colorOffset,
+            in_defaultColor,
+            in_alphaGain,
+            in_alphaOffset,
+            in_invert,
+            in_alphaIsLuminance,
+            out_outColor,
+            out_outAlpha);
+
+        return;
+    }
+    
+    if (in_uvCoord[0] == OUTSIDE_UVFRAME || in_uvCoord[1] == OUTSIDE_UVFRAME)
+    {
+        out_outColor = in_defaultColor;
+        out_outAlpha = maya_luminance(in_defaultColor);
+    }
+    else
+    {
+        int channels = 0;
+        int size[2] = {0,0};
+
+        gettextureinfo(in_fileTextureName, "channels", channels);
+        gettextureinfo(in_fileTextureName, "resolution", size);
+
+        out_outSize[0] = size[0];
+        out_outSize[1] = size[1];
+
+        out_fileHasAlpha = (channels > 3) ? 1 : 0;
+
+        float st[2] = {in_uvCoord[0], in_uvCoord[1]}, dsdt[4];
+
+        string filter = "";
+
+        if (in_filterType)
+        {
+            dsdt[0] = Dx(st[0]);
+            dsdt[1] = Dx(st[1]);
+            dsdt[2] = Dy(st[0]);
+            dsdt[3] = Dy(st[1]);
+
+            // Starting point for filter choice (rough) matching.
+
+            if (in_filterType == 1)
+            {
+                filter = "linear";
+            }
+            else if (in_filterType == 2)
+            {
+                filter = "closest";
+            }
+            else if (in_filterType == 3)
+            {
+                filter = "cubic";
+            }
+            else if (in_filterType == 4)
+            {
+                filter = "smartcubic";
+            }
+            else if (in_filterType == 5)
+            {
+                filter = "smartcubic";
+            }
+            else
+            {
+                filter = "closest";
+            }
+        }
+        else
+        {
+            dsdt[0] = 0;
+            dsdt[1] = 0;
+            dsdt[2] = 0;
+            dsdt[3] = 0;
+            
+            filter = "closest";
+        }
+
+        // Disabled filtering can still be overriden by filter radius.
+
+        if (in_preFilter)
+        {
+            float fw = in_filterWidth * in_preFilterRadius;
+
+            dsdt[0] = fw;
+            dsdt[1] = fw;
+            dsdt[2] = fw;
+            dsdt[3] = fw;
+        }
+
+        if (in_filterType == 1 && in_useMaximumRes)
+        {
+            out_outColor =
+                texture(
+                    in_fileTextureName,
+                    st[0], st[1],
+                    dsdt[0], dsdt[1], dsdt[2], dsdt[3],
+                    "blur", in_filterOffset,
+                    "width", in_filter,
+                    "wrap", "periodic",
+                    "subimage", 0,
+                    "missingcolor", in_defaultColor,
+                    "missingalpha", 1,
```

Comment:
@luisbarrancos Can you change it by "missingalpha", 1.0?
It seems that for keyword arguments, you need to use the exact type for the values.